models/topics.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection')
+function fetchAllTopics(){
+    return db.query('SELECT * FROM topics')
+    .then(({rows})=> {
+        return rows
+    })
+    .catch((err)=>{
+        return Promise.reject(err)
+    })
```

Comment:
A catch is not needed in the model, if this promise was to fail/reject then the catch in your controller would catch it.